### PR TITLE
add category routing

### PR DESCRIPTION
**Pull request by srhboo**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -28,8 +28,10 @@ const history = syncHistoryWithStore(browserHistory, store);
 const ReduxApp = () => (
   <Provider store={store}>
     <Router history={history}>
-      <Route path="/" component={App}/>
-      <Route path="/foo" component={Foo}/>
+      <Route path="/" component={App}>
```
Comment (auredufour) [inline src/index.js]: I think it should be {browserHistory} instead of {history}

Comment (turquoisemelon) [inline src/index.js]: +1 to @auredufour 's comment

**Comment by turquoisemelon** [comment]
Great job! Should be good to merge after adding the browserHistory.

File: src/index.js (hunk 1 above)
Comment (srhboo) [inline src/index.js]: @auredufour @turquoisemelon if you expand the code for this page and look at line 26, browserHistory is synced with store to make the history variable. It's from the code here I think we were shown in class >> https://github.com/reactjs/react-router-redux. Thoughts?

**Pull request merged**